lib/src/soloud.dart

```diff
@@ -202,6 +203,7 @@ interface class SoLoud {
   /// unnecessary, as the amount of data will be finite.
   /// The default is `false`.
   Future<void> init({
+    // TODO(filip): remove deprecation?
```

Comment:
we forgot to take a look at this deprecation.

Reply:
Thanks for pointing it out! I'll have a look in a different PR, this one is already large.